pkg/sql/pgwire/testdata/pgtest/param_status

```diff
@@ -17,15 +17,7 @@ send
 Query {"String": "SET TIME ZONE +6"}
 ----
 
-until crdb_only
```

Comment:
give this a go with PG (`go test ./pkg/sql/pgwire -run TestPGTest/param_status -rewrite -addr='localhost:5432' --user=otan`) to double check :)

Reply:
It worked!